[2022-03-21 16:00:34,127] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: test_postgress.python_task 2022-03-21T16:00:33.660296+00:00 [queued]>
[2022-03-21 16:00:34,145] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: test_postgress.python_task 2022-03-21T16:00:33.660296+00:00 [queued]>
[2022-03-21 16:00:34,145] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-03-21 16:00:34,145] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-03-21 16:00:34,145] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-03-21 16:00:34,155] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): python_task> on 2022-03-21T16:00:33.660296+00:00
[2022-03-21 16:00:34,163] {standard_task_runner.py:52} INFO - Started process 1493 to run task
[2022-03-21 16:00:34,167] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'test_postgress', 'python_task', '2022-03-21T16:00:33.660296+00:00', '--job-id', '633', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/test_postgress.py', '--cfg-path', '/tmp/tmp1navfb0j', '--error-file', '/tmp/tmpajbdtwsl']
[2022-03-21 16:00:34,167] {standard_task_runner.py:77} INFO - Job 633: Subtask python_task
[2022-03-21 16:00:34,209] {logging_mixin.py:104} INFO - Running <TaskInstance: test_postgress.python_task 2022-03-21T16:00:33.660296+00:00 [running]> on host e39215f0be5b
[2022-03-21 16:00:34,299] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test_postgress
AIRFLOW_CTX_TASK_ID=python_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-21T16:00:33.660296+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-03-21T16:00:33.660296+00:00
[2022-03-21 16:00:34,299] {taskinstance.py:1455} ERROR - _select_all_platforms() missing 1 required positional argument: 'sql_file'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1112, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1285, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1315, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/python.py", line 117, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/python.py", line 128, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
TypeError: _select_all_platforms() missing 1 required positional argument: 'sql_file'
[2022-03-21 16:00:34,301] {taskinstance.py:1503} INFO - Marking task as FAILED. dag_id=test_postgress, task_id=python_task, execution_date=20220321T160033, start_date=20220321T160034, end_date=20220321T160034
[2022-03-21 16:00:34,380] {local_task_job.py:146} INFO - Task exited with return code 1
